supabase/migrations/20231017171314_my_table.sql

```diff
@@ -0,0 +1,54 @@
+drop policy "Enable read access for authenticated users." on "public"."pallet";
+
+drop policy "Enable insert for authenticated users only" on "public"."project";
+
+drop policy "Enable read access for authenticated users." on "public"."shipment";
+
+alter table "public"."pallet" disable row level security;
+
+alter table "public"."pallet_scan" disable row level security;
+
+alter table "public"."project" disable row level security;
+
+alter table "public"."shipment" add column "bookkeeping_flexport" boolean;
+
+alter table "public"."shipment" add column "bookkeeping_gik" boolean;
+
+alter table "public"."shipment" add column "commercial_invoice_link" text;
+
+alter table "public"."shipment" add column "consignee" text;
+
+alter table "public"."shipment" add column "destination" text not null;
+
+alter table "public"."shipment" add column "doner" text;
```

Comment:
Did we get to the bottom of whether we will have a specific list of these people? Might be good again to have a `doners` table?